test/spmd/test_xla_sharding.py

```diff
@@ -638,15 +580,20 @@ def test_2d_tensor_3d_mesh(self):
 
     t1 = ct1.to(xm.xla_device())
     t2 = ct2.to(xm.xla_device())
-    mesh = self._get_mesh((1, self.n_devices, 1))
-    t1 = xs.mark_sharding(t1, mesh, partition_spec=(1, 2))
+
```

Comment:
cc @khatwanimohit @jonb377 I updated this unit test, could you confirm that this would be relevant for the multi-slice test cases, also note that in case of 1-device, it falls back to replicated.

Reply:
Yes, this is how we shard model parameters in shardings.py